src/displayapp/screens/WatchFaceTerminal.cpp

```diff
@@ -38,10 +38,11 @@ WatchFaceTerminal::WatchFaceTerminal(DisplayApp* app,
 
   connectState = lv_label_create(lv_scr_act(), nullptr);
   lv_label_set_recolor(connectState, true);
+  lv_label_set_text(connectState, "[STAT]#0082fc Disconnected#");
```

Comment:
I removed this line in #1002, because this gets overridden later in the call to `Refresh()`, making this initial text unnecessary. I think it came back during the merging. It should be safe to remove.